### ISSUE TITLE
accessibility statement and feedback mechanism

### ISSUE THREAD

**Issue by whuml** [moodle-an-hochschulen/moodle-theme_boost_union#567]
As already diskussed we suggest following enhancement(s) (feature) for Boost Union to improve accessibility resp. provide important accessibilty functionalities:

# accessibility statement
To be able to comply with EU Directive 2016/2102 
https://eur-lex.europa.eu/legal-content/DE/TXT/PDF/?uri=CELEX:32016L2102 
there should be the possibility for universities to offer an **accessibility statement** within Moodle, respectively with the BoostUnion theme. 
In addition to the customizable description, the accessibility statement should contain a link to electronic contact, via which users can report any deficiencies in Moodles course pages, in terms of compliance with accessibility requirements, to the relevant universities office (which can be defined by the Moodle admins in the settings) [...]." ($9b NBGG)

### Minimum requirements:
The accessibility statement and feedback mechanism are easy to find (question mark, footer button, footer)
The accessibility statement describes and directs to the feedback mechanism.
The accessibility statement and the feedback mechanism itself must be accessible. 

### sample:
![grafik](https://github.com/moodle-an-hochschulen/moodle-theme_boost_union/assets/104976983/5276137e-4606-441f-868c-e6af321ae890)

# accessibility reporting button
It should be possible for admins to activate a further footerbutton, "**accessibility reporting button**" and/or footer link to feedback mechanism.

### Minimum requirements of the feedback mechanism:
The feedback mechanism (button) should be placed directly above the Moodle question mark (footer button) and should always be accessible. There should be the option to decide whether an aditional feedback link should be placed within the footer.

### sample:
![grafik](https://github.com/moodle-an-hochschulen/moodle-theme_boost_union/assets/104976983/026ec004-b655-40e0-a4f8-3fb11a692da2)

### Mechanism provides automated text:
- Technical data such as operating system, browser used, etc. are pre-filled.
- The URL of the page with the barrier is transmitted automatically
- Form to answer Question if User works with assistive technologies such as a screen reader, zoom text,
   voice control or similar? If yes, please specify which ones.

# single feedback page
All contact options should be available on that single feedback page:
- Call accessibility support
- Mail accessibility support
- Form to contact accessibility support

### sample:
![grafik](https://github.com/moodle-an-hochschulen/moodle-theme_boost_union/assets/104976983/bdd5239a-00ab-4f0a-b52f-61a9950b73e9)

# 

### admin settings static page sample:
![grafik](https://github.com/moodle-an-hochschulen/moodle-theme_boost_union/assets/104976983/703700d9-3cd9-4a9f-9067-0bfcfcbd89e7)
# 
### Many thanks in advance for boostunion teams efford to further improve accessibility :-)
